<launch>

<rosparam command="load" file="$(find roboteq_motor_controller_driver)/config/query.yaml" />

<node name = "roboteq_motor_controller_driver" pkg = "roboteq_motor_controller_driver" type = "roboteq_motor_controller_driver_node" output = "screen">


</node>

 
</launch>
